gui/default/syncthing/settings/settingsModalView.html

```diff
@@ -142,7 +142,8 @@
             <div ng-if="tmpOptions.upgrades != 'candidate'">
               <label translate for="urVersion">Anonymous Usage Reporting</label> (<a href="" translate data-toggle="modal" data-target="#urPreview">Preview</a>)
               <select class="form-control" id="urVersion" ng-model="tmpOptions._urAcceptedStr">
-                <option ng-repeat="n in urVersions()" value="{{n}}">Version {{n}}</option>
+                  <option ng-repeat="n in urVersions()" value="{{n}}">{{'Version' | translate}} {{n}}</option>
+                  <!-- <option ng-repeat="n in urVersions()" value="{{n}}"><translate>Version</translate> {{n}}</option> -->
                 <!-- 1 does not exist, as we did not support incremental formats back then. -->
```

Comment:
Whats up with the comments?

Reply:
And indent?

Reply:
No idea what it is with the comment, not of my doing, I didn't even notice before. Indenting is my wrongly adjusted editor -> will fix.